front/app/components/FilterBoxes/InputFilterCollapsible.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import {
+  Box,
+  CollapsibleContainer,
+  colors,
+} from '@citizenlab/cl2-component-library';
+
+interface Props {
+  title: string;
+  children: JSX.Element;
+  className?: string;
+}
+
+const InputFilterCollapsible = ({ title, children, className }: Props) => {
+  return (
+    <Box
+      className={className}
+      background={colors.white}
+      borderRadius="3px"
+      mb="20px"
+      p="12px"
+    >
```

Comment:
Hmm, we shouldn't have to do this. Styles should be configurable via CollapsibleContainer's props. But it's not super straightforward. I'll try some things.